torch/optim/lr_scheduler.py

```diff
@@ -700,6 +700,76 @@ def load_state_dict(self, state_dict):
             self._schedulers[idx].load_state_dict(s)
 
 
+class PolynomialLR(_LRScheduler):
+    """Decays the learning rate of each parameter group using a polynomial function,
+    to reach a min_lr in the given total_iters. When last_epoch=-1, sets initial lr as lr.
+
+
+    Args:
+        optimizer (Optimizer): Wrapped optimizer.
+        total_iters (int): After this step, learning rate stops decreasing.
+        min_lr (float or list): A scalar or a list of scalars. A
+            lower bound on the learning rate of all param groups
+            or each group respectively. Default: 0.
+        power (int): The power of the polynomial. Defaults to linear, 1.0.
```

Comment:
```suggestion
        power (int): The power of the polynomial. Default: ``1.0``.
```

Reply:
Why the default values here is ``1.0``, while `total_iters` is just `5`?

Reply:
Thanks for pointing out. It seems that PyTorch Core doesn't use backticks for numerical values. Feel free to skip them.

Reply:
Done.